librz/core/cmd_descs/cmd_flag.yaml

```diff
@@ -150,4 +341,10 @@ commands:
           - RZ_OUTPUT_MODE_STANDARD
           - RZ_OUTPUT_MODE_JSON
         args: []
+  - name: fx
+    cname: flag_hexdump
+    summary: Show hexdump of flag:flagsize
+    args:
+      - name: old
```

Comment:
change name to this

Reply:
To this? What do you mean?

Reply:
I mean replace `old` with something else :D

Reply:
Replace `old` with `young`?

Reply:
:D no it's just that `old` does not make sense in this context IMO. The command just shows the hexdump of a flag with a give flagname, so there is no "old".